tests/unit/_experimental/classification/test_utils.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2021-2023 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+import pytest
+
+from kolena.workflow.annotation import ScoredClassificationLabel
+from kolena.workflow.plot import Histogram
+
+
+@pytest.mark.parametrize(
+    "test_name, label, inference_labels, expected",
+    [
+        ("empty", "", [], 0),
+        ("one diff", "a", [ScoredClassificationLabel("b", 0.1)], 0),
+        ("one same", "a", [ScoredClassificationLabel("a", 0.1)], 0.1),
+        (
+            "many diff",
+            "a",
+            [
+                ScoredClassificationLabel("b", 0.1),
+                ScoredClassificationLabel("c", 0.2),
+                ScoredClassificationLabel("d", 0.3),
+                ScoredClassificationLabel("e", 0.4),
+            ],
+            0,
+        ),
+        (
+            "full",
+            "a",
+            [
+                ScoredClassificationLabel("c", 0.1),
+                ScoredClassificationLabel("b", 0.2),
+                ScoredClassificationLabel("a", 0.3),
+                ScoredClassificationLabel("d", 0.4),
+            ],
+            0.3,
+        ),
+    ],
+)
+def test__get__label__confidence(
+    test_name: str,
+    label: str,
+    inference_labels: List[ScoredClassificationLabel],
+    expected: float,
+) -> None:
+    from kolena._experimental.classification.utils import get_label_confidence
```

Comment:
Curious why you are importing within the scope of this function instead of importing at the top

Reply:
`importorskip`, which I didn't know of before, is a workaround to the issue of module imports that reply on the sklearn extra. There's no way to `@pytest.mark.metrics` an import, so in the past, my workaround was to put it inside of each test.